packages/v4/src/content/get-started/design.md

```diff
@@ -7,24 +7,33 @@ import './get-started.css';
 
 Whether you are converting from a PatternFly 3 application or starting fresh with the latest and greatest PatternFly design system, this document is intended to help provide designers with information regarding resources and support available.
 
-
 ## Getting the design kit
 
-### Before you begin
-The design kit is a collection of Sketch assets that make it easy for designers to create hi-fi mockups that accurately represent PatternFly components and layouts. It works with Sketch libraries.
+The PatternFly design kit was originally developed for [Sketch users](#Mac-users), but is now available for use through [any design application](#Non-mac-users).
```

Comment:
the "any design application" link doesn't work because the "non" in "non mac users" needs to be sentence case!